react/components/EditorContainer/Sidebar/utils.test.ts

```diff
@@ -140,6 +150,78 @@ describe('getComponents', () => {
     ])
   })
 
+  describe('schema with no titles', () => {
+    consoleWarnSetup()
+
+    it('should show extension that have only title in the blocks (extensions) and no schema', () => {
+      expect(
+        getComponents(
+          {
+            'store/home': {
+              component: 'vtex.LayoutContainer',
+              props: {
+                elements: ['header', 'layout'],
+              },
+            },
+            'store/home/header': {
+              component: 'vtex.schema-without-title',
```

Comment:
⛏ Perhaps `vtex.schema-with-title`?

Reply:
The component schema doesn't have a title... In fact, there isn't even a component, so I guess I could change that.